uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,64 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+
+  return page.extend({
+
+    /**
+     * [openMyReport description]
+     * @return {[Page]} [description]
+     */
+    openMyReport: function () {
+      var casper = this.casper;
+
+      casper.then(function () {
+        casper.waitUntilVisible('td[ng-click="report.toggleSection(\'pending\')"]', function () {
+          casper.wait(700);
```

Comment:
Yes, but when the load is complete some element should be Visible. We can waitUntil that.